src/resize.cc

```diff
@@ -828,8 +842,8 @@ class ResizeWorker : public NanAsyncWorker {
       // Info Object
       Local<Object> info = NanNew<Object>();
       info->Set(NanNew<String>("format"), NanNew<String>(baton->outputFormat));
-      info->Set(NanNew<String>("width"), NanNew<Integer>(width));
-      info->Set(NanNew<String>("height"), NanNew<Integer>(height));
+      info->Set(NanNew<String>("width"), NanNew<Uint32>(static_cast<uint32_t>(width)));
```

Comment:
Thanks for future-proofing this conversion to handle much larger values - good spot!